parameters.yml

```diff
@@ -174,11 +174,13 @@ asciidoc:
   - attribute: sp-base
     value: SP4
   - attribute:  sp-version
-    value: SP5 #use for normal text
+    value: SP4 #use for normal text
   - attribute: sp-vert
```

Comment:
I wonder why we need `sp-vert` and `sp-version-l` if the value is 100% the same but... OK :-D

Reply:
> I wonder why we need `sp-vert` and `sp-version-l` if the value is 100% the same but... OK :-D

long time ago, I defined `sp-version-l`.  It some incarnation of `sp-vert` or `sp-ver` in long format.  I do not remember exactly the values.  Maybe, "SP 5" vs. "Service Pack 5" or just "5" vs. "SP 5" ;)  We often do not know how to properly phase out a variable no longer needed.

Reply:
lol not sure why I did not notice this before. We should keep these simple for sure. Phase it out! Phase it out! :)